package.json

```diff
@@ -35,23 +42,14 @@
     "eslint": "^8.47.0",
     "eslint-config-braintree": "^6.0.0-typescript-prep-rc.2",
     "eslint-plugin-prettier": "^5.0.0",
-    "jest": "^29.6.3",
-    "jest-environment-jsdom": "^29.6.3",
+    "happy-dom": "^10.11.2",
```

Comment:
what's happy-dom and why are we using it?

Reply:
It's a JS implementation of a web browser (mocking one essentially). Can use this or [jsdom](https://www.npmjs.com/package/jsdom).